README.md

```diff
@@ -71,7 +71,7 @@ in a wide variety of other scientific simulations, some of which, can be seen
 in our application [gallery](https://amrex-codes.github.io/amrex/gallery.html).
 
 <div align="center">
-<img src="https://github.com/etpalmer63/etpalmer63.github.io/blob/f07ae02daf5f58d25c4cdad712f4024c62ea50b7/Images/gallery.gif" alt="Gallery Slideshow" style="width:500px">
+<img src="https://github.com/amrex-codes/amrex-codes.github.io/blob/main/Images/gallery_small.gif" alt="Gallery Slideshow">
```

Comment:
```suggestion
<img src="https://github.com/amrex-codes/amrex-codes.github.io/blob/main/images/gallery_small.gif" alt="Gallery Slideshow">
```
otherwise I get a 404.